### DISCUSSION TITLE
react-upload

### DISCUSSION THREAD

**Discussion by toprater** [community/community#45726] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

I made a input form to upload user's photo and documents but it looked very slow when user drags the item.
Is there any specialized npm module for this kind of work?

**Comment by Juiux** [comment, marked as answer]
You can user these below modules for your work.

https://www.npmjs.com/package/drag-drop
https://www.npmjs.com/package/ngx-drag-drop
https://www.npmjs.com/package/react-drag-and-drop

I suggest you to use NgxDragDrop as I built the same work with this one and now, it's working properly.
